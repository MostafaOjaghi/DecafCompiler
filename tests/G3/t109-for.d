void main() {
	int i;
	for (i = 0; i < 10; i = i + 1) {
		int a;
		a = 2 * i + 1;
		Print(a,i);
	}
	i = 0;
	for (;i < 10;) {
		int a;
		a = 2 * i + 1;
		Print(a,i);
		i = i + 1;
	}
}
