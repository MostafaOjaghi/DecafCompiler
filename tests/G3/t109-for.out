10
31
52
73
94
115
136
157
178
199
10
31
52
73
94
115
136
157
178
199
